src/ess/livedata/config/instruments/dream/factories.py

```diff
@@ -45,6 +45,13 @@ def _get_wire_view(da: sc.DataArray) -> sc.DataArray:
     )
 
 
+def _get_strip_view(da: sc.DataArray) -> sc.DataArray:
+    """Transform function to extract strip view (sum over all but strip)."""
+    return da.fold(dim=da.dim, sizes=_bank_sizes['mantle_detector']).sum(
+        ('wire', 'module', 'segment', 'counter')
```

Comment:
Why does this one sum over the other dims but the wire view flattens them?

Reply:
This view corresponds to the `z` in the mantle. The other view has `(r, phi)` for `r~wire` and `phi` is the flattened other dims. Together these two views give a view of all readout components.

Reply:
I think this will mainly be for IS/IDS to identify broken components. But obviously one will have to iterate on titles and description. My current plan is to source those from the IS/IDS after we have converged on what they want plotted.

Reply:
If a counter breaks (I don't know if it can independently) you will see a particular pattern in the wire view, as far as I understand.

The scientists asked for seeing all voxels, but that is not reasonable in streaming. Therefore I pushed for an imho more usable solution, one that shows all independent readout components. Here is a simpler example: On instruments with He3 tubes there is a single wire per tube connected to readout. Therefore, either the entire wire is faulty or not. For diagnostics it should thus be sufficient to show a plot that has one pixel (or value) per wire. Same concept here, except that there is an extra dimension, with 2 sets of orthogonaly arranged components, so we have two plots.